api/envoy/api/v2/route/route.proto

```diff
@@ -1076,6 +1076,20 @@ message Tracing {
   // :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
   // Default: 100%
   envoy.type.FractionalPercent overall_sampling = 3;
```

Comment:
To reviewers, i have no idea where to place this for common use, duplicated here.
Do you have any thoughts? thanks.